.travis.yml

```diff
@@ -31,11 +27,21 @@ addons:
       - google-chrome-stable
       - g++-4.8
 
-# NPM Canary Build Config
-deploy:
-  skip_cleanup: true
-  provider: script
-  script: yarn release --canary
-  on:
-    repo: firebase/firebase-js-sdk
-    branch: master
+jobs:
+  include:
+    - script: xvfb-run yarn test
+      after_success: yarn test:coverage
+      env: '#comment: yarn test'
+    - script: xvfb-run yarn test:saucelabs
+      env: '#comment: yarn test:saucelabs'
```

Comment:
So I think we want to just completely disable this stage in the event that it is not a push build, I just realized that we run the same suite twice differently in two browsers and that seems unnecessary.

See: https://docs.travis-ci.com/user/conditional-builds-stages-jobs/ for an easy way of doing this.

Reply:
Done.

I'm using env as a description for these two Travis jobs. It turns out that for if condition to work, there has to be a '=' in the env statement.